.gitmodules

```diff
@@ -1,4 +1,6 @@
+[submodule "cloudomate"]
+	path = cloudomate
+	url = https://github.com/Tribler/cloudomate.git
 [submodule "tribler"]
 	path = tribler
-	url = https://github.com/rjwvandenberg/tribler
-	branch = plebnet
+	url = https://github.com/HvanVeltom/tribler.git
```

Comment:
Please point this to the right repository (the official Tribler repo)

Reply:
PlebNet uses an own version of the twisted plugin for starting Tribler headless, allowing it to start Tribler as an exitnode and/or with testnet